src/parser/vast_parser.js

```diff
@@ -351,20 +368,39 @@ export class VASTParser extends EventEmitter {
       wrapperSequence = null,
       previousUrl = null,
       wrapperDepth = 0,
-      isRootVAST = false
-    }
+      isRootVAST = false,
+      followAdditionalWrappers,
+      allowMultipleAds
+    } = {}
   ) {
     let ads = [];
+    // allowMultipleAds was introduced in VAST 3 as wrapper attribute
+    // for retrocompatibility set it to true for vast pre-version 3
+    if (this.vastVersion && parseFloat(this.vastVersion) < 3 && isRootVAST) {
+      allowMultipleAds = true;
+    }
     try {
       ads = this.parseVastXml(vastXml, {
         isRootVAST,
         url,
-        wrapperDepth
+        wrapperDepth,
+        allowMultipleAds,
+        followAdditionalWrappers
       });
     } catch (e) {
       return Promise.reject(e);
     }
 
+    // if wrapper allowMultipleAds is set to false only the first stand-alone Ad
+    // (with no sequence values) in the requested VAST response is allowed
+    if (
+      typeof allowMultipleAds === 'boolean' &&
```

Comment:
We can probably replace `typeof allowMultipleAds === 'boolean' && !allowMultipleAds` by` allowMultipleAds === false`